core/src/main/java/nl/nn/adapterframework/configuration/Configuration.java

```diff
@@ -423,6 +431,9 @@ public void registerAdapter(Adapter adapter) {
 	}
 
 	// explicitly in this position, to have the right location in the XSD
+	/**
+	 * Container for jobs scheduled for periodic execution.
+	 */
```

Comment:
De manager is niet de container. De javadoc van registerScheduledJob kan/moet gebruikt worden op het 'scheduler' element.

Reply:
This will be improved by implementing https://github.com/ibissource/frank-doc/issues/126.